crates/ty_python_semantic/src/types/call/bind.rs

```diff
@@ -650,6 +653,14 @@ impl<'db> Bindings<'db> {
                             None
                         };
 
+                        let alias = alias.and_then(|alias_ty| {
+                            if let Type::StringLiteral(s) = alias_ty {
+                                Some(s.value(db).to_string().into_boxed_str())
+                            } else {
+                                None
+                            }
+                        });
```

Comment:
```suggestion
                        let alias = alias
                            .and_then(Type::as_string_literal)
                            .map(|literal| Box::from(literal.value(db)));
```

(though this would need to be revised slightly if you do refactor to use `Option<Name>` instead of `Option<Box<str>>` 😄